maindata.py

```diff
@@ -4,32 +4,44 @@
 from csvoutput import CsvOutput
 from wikipediafunction import WikipediaFunction
 from youtubefunction import YoutubeFunction
+from steamfunction import SteamFunction
+#import time
+from multiprocessing.pool import ThreadPool
+
+pool = ThreadPool(processes=1)
 
 redditFunction = RedditFunction()
 imdbFunction = ImdbFunction()
 csvOutput = CsvOutput()
 wikipediaFunction = WikipediaFunction()
 youtubeFunction = YoutubeFunction()
+steamFunction = SteamFunction()
 
 #input string
 q = 'gravity'
 
-#adapt string for reddit or imdb query
-queryr = q.replace(" ","+")
-queryi = q.replace(" ","-")
-queryw = q.replace(" ","+")
-queryy = q.replace(" ","+")
+#tic = time.perf_counter()

```

Comment:
Love that you are measuring performance and using threading for optimizations! 🥳